### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JoeZiminski**
Add some installation instructions. Should we link to somewhere that tells people how to setup a condo environment?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,17 @@ It will also set up:
...
-First, install cookiecutter in your desired environment.
+### Installating Cookiecutter
+
+First, install cookiecutter in your desired environment. Running in the terminal in your environment, with Pip:
+
+`pip install cookiecutter`
+
+or conda:
+
+`conda install cookiecutter`
```
Comment (adamltyson) [inline README.md:47]: Should this come from the main channel or conda-forge?

**Review by adamltyson — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (adamltyson) [reply README.md:47]: @JoeZiminski bump

Comment (JoeZiminski) [reply README.md:47]: should come from conda forge, is:

`conda install -c conda-forge cookiecutter`

or

```
conda config --add channels conda-forge
conda install cookiecutter
```
better to recommend?

Comment (alessandrofelder) [reply README.md:47]: my preference would be the former. While the latter is more explicit, my expectation is that advanced users are familiar with the former, and new users of `conda` won't really understand either, so we should give them one fewer line to copy-paste?

Comment (adamltyson) [reply README.md:47]: No idea really. Maybe we want to say `conda install -c conda-forge cookiecutter`. Here probably isn't the place to start a lesson about conda channels?

**Pull request merged**